src/Root.tsx

```diff
@@ -0,0 +1,29 @@
+import {
+  HashRouter as Router,
+  Navigate,
+  Route,
+  Routes,
+} from 'react-router-dom';
+import { App } from './App';
+import { HomePage } from './Pages/HomePage';
+import { PeoplePage } from './Pages/PeoplePage';
+import { NotFoundPage } from './Pages/NotFoundPage';
+
+export const Root = () => {
+  return (
+    <Router>
+      <Routes>
+        <Route path="/" element={<App />}>
+          <Route index element={<HomePage />} />
+          <Route path="people" element={<PeoplePage />}>
+            <Route path=":slug" element={<PeoplePage />} />
+          </Route>
+
+          <Route path="/home" element={<Navigate to="/" replace />} />
+          <Route path="*" element={<NotFoundPage />} />
+        </Route>
+      </Routes>
+      <App />
```

Comment:
The `<App />` component is rendered twice: once as a route element and once outside the `<Routes>` block. This redundancy could cause issues. Consider removing the second instance of `<App />` on this line.